tests/snappi_tests/multidut/pfc/files/multidut_helper.py

```diff
@@ -229,12 +230,14 @@ def run_pfc_test(api,
 
     # Clear PFC, queue and interface counters before traffic run
     duthost = egress_duthost
-    duthost.command("pfcstat -c")
-    time.sleep(1)
-    duthost.command("sonic-clear queuecounters")
-    time.sleep(1)
-    duthost.command("sonic-clear counters")
-    time.sleep(1)
+
+    for dut in [egress_duthost, ingress_duthost]:
+        dut.command("pfcstat -c")
+        time.sleep(1)
+        dut.command("sonic-clear queuecounters")
+        time.sleep(1)
+        dut.command("sonic-clear counters")
+        time.sleep(1)
```

Comment:
@sdszhang These are the changes needed to fix the test_pfc_pause_counter_check on T2. I have double check on t0 it still pass as well